.gitignore

```diff
@@ -1,34 +1,28 @@
+# python
 *.pyc
 *.pyo
 *.swp
-*.html
-*.class
-*.orig
-*.rej
-*~
 __pycache__
 
+
+# packaging folders
 src/tox/version.py
+build/
+dist/
+src/tox.egg-info
```

Comment:
another tip so you don't lose your files again, you can prefix these with a `/` to just cover the root of the repository [for instance](https://github.com/asottile/pyupgrade/blob/1db939202cdafbd1c144e165157cd4fffc8520c9/.gitignore#L3-L6)

Reply:
Hmm, I thought to put the / at end will have the same effect. Not?

Reply:
And yes this is only moving code around.

Reply:
nope!

```console
$ tree .
.
├── build
│   └── ignore_me
└── t
    └── build
        └── dont_ignore_me

3 directories, 2 files
$ cat .gitignore 
build/
$ git status
On branch master

No commits yet

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	.gitignore

nothing added to commit but untracked files present (use "git add" to track)
```